package_control/package_manager.py

```diff
@@ -1292,8 +1292,9 @@ def install_package(self, package_name, is_dependency=False):
 
             # Here we don't use .extractall() since it was having issues on OS X
             overwrite_failed = False
-            extracted_paths = []
-            for path in package_zip.namelist():
+            extracted_paths = set()
+            for info in package_zip.infolist():
+                path = info.filename
                 dest = path
```

Comment:
*Could* be shortened to `dest = path = info.filename`, but really does't matter.